DigitalLearningSolutions.Web/ViewModels/LearningMenu/InitialMenuViewModel.cs

```diff
@@ -1,14 +1,18 @@
 ﻿namespace DigitalLearningSolutions.Web.ViewModels.LearningMenu
 {
+    using System.Collections.Generic;
+    using System.Linq;
     using DigitalLearningSolutions.Data.Models.CourseContent;
 
     public class InitialMenuViewModel
     {
         public CourseContent CourseContent { get; }
+        public IEnumerable<SectionCardViewModel> Sections { get; }

```

Comment:
The sections are already part of the CourseContent so I don't think we should also store them in an additional variable. We can instead add a helper method to the model for getting the percentage complete 🙂

Reply:
I mainly did this to follow what had been done for the course cards. If I were to remove this, how would I pass the specific section to the partial cshtml I have made? As if I just pass `InitialMenuViewModel` it won't know which section to use. The other alternative is to have the foreach loop within the partial class and then it would work with `InitialMenuViewModel`. What do you think is the best way?

Reply:
Ah I see. In that case I would keep the section card view model but remove the CourseContent variable and instead store the parts of the course content that you need. Does that make sense? 🙂